cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/analyzer/kafkaassigner/KafkaAssignerDiskUsageDistributionGoal.java

```diff
@@ -37,8 +44,8 @@
 public class KafkaAssignerDiskUsageDistributionGoal implements Goal {
   private static final Logger LOG = LoggerFactory.getLogger(KafkaAssignerDiskUsageDistributionGoal.class);
   private static final double BALANCE_MARGIN = 0.9;
-  // If broker disk usage differ less than BROKER_EQUALITY_DELTA, consider them as equal -- i.e. no swap.
-  private static final double BROKER_EQUALITY_DELTA = 1.0;
+  // If broker disk usage differ less than USAGE_EQUALITY_DELTA, consider them as equal -- i.e. no swap.
+  private static final double USAGE_EQUALITY_DELTA = 0.0001;
```

Comment:
While it is desirable to achieve an even balance, I am worried that this level of requirement will lead to excessive swaps. Do we really need up to 100-bytes evenness in disk usage? I would propose to apply a subset of the following 1) increase `USAGE_EQUALITY_DELTA`, 2) set a timeout, 3) set a limit on the number of iterations.

Reply:
The usage is in percentage. So 0.0001 of a 1TB disk would be about 100MB difference.